include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -85,8 +85,8 @@ class BrushedMotorInterruptHandler {
     }
 
     void homing_stopped() {
-        finish_current_move(AckMessageId::stopped_by_condition);
         hardware.reset_encoder_pulses();
+        finish_current_move(AckMessageId::stopped_by_condition);
```

Comment:
shouldn't this be before resetting the encoder pulses?

Reply:
We want to reset first because if this is the first time the motor homes after boot up, the encoder position could be an extremely large negative value.